src/pages/workspaces/workspace/Data.js

```diff
@@ -563,17 +566,35 @@ const WorkspaceData = _.flow(
     }
   }
 
-  const loadWdsSchema = async () => {
-    if (isFeaturePreviewEnabled('workspace-data-service') && !getConfig().isProd) {
-      try {
-        setWdsSchema([])
-        setWdsSchemaError(undefined)
-        const wdsSchema = await Ajax(signal).WorkspaceData.getSchema(workspaceId)
-        setWdsSchema(wdsSchema)
-      } catch (error) {
-        setWdsSchemaError(error)
+  const loadWdsSchemaAndUrl = async () => {
+    await Ajax(signal).Apps.getV2AppInfo(workspaceId).then(async apps => {
+      // TODO: Update to pull specifically from /api/apps/v2/{workspaceId}/{appName}
+      if (isFeaturePreviewEnabled('workspace-data-service') && !getConfig().isProd) {
```

Comment:
@davidangb just noticed this `getConfig().isProd` flag -- is this related to the presence of the fact this is Feature Preview, or is there something else we need to code regarding this conditional?

Reply:
nothing to do here; this is just ensuring the preview doesn't leak out before we want it to. AJ-688 will remove this.